ui/assets/img/filter-icon.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Similar: do we want to include these icons for some future date or keep them out of the code base for now?